lib/src/rules/length.ts

```diff
@@ -0,0 +1,29 @@
+export interface LengthParams {
+  length: number;
+}
+
+export function parseLengthParams(customParams: LengthParams, errorMessage: string) {
+  const length = customParams.length === null ? NaN : Number(customParams.length);
```

Comment:
Why do you have to parse `customParams.length` to number if `LengthParams.lenght` field is a number?